src/WarcraftLegacies.Source/Quests/Scourge/QuestPlague.cs

```diff
@@ -93,6 +93,10 @@ protected override void OnComplete(Faction completingFaction)
       completingFaction.Player.RescueGroup(_coastUnits);
       completingFaction.Player.RescueGroup(_scholomanceUnits);
 
+      CreateUnit(_secondaryPlagueFaction.Player, Constants.UNIT_U005_DREAD_SHRINE_LEGION_SPECIAL, 11138, 12802, 0);
```

Comment:
Is this conveyed to either Scourge or Legion at the moment? Possibly need to think about making this clearer. Legion might need a quest that completes when the Plague is completed just for this.

Reply:
Right now Alterac complètes with the plague.  Can add those to it!

Reply:
Interesting... My follow-up question, then, is: how does the player know that the Alterac quest completes with the Plague? The quest doesn't say that, right?

Reply:
Hum, no directly I guess no.

Reply:
OK, the objective for that really needs to be "Scourge has completed Plague of Undeath". That's easy enough for me to do - I think I had it in place for Forsaken before their removal. I'll do that in another PR: https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2472

So yeah, to resolve this you just need to move the Dread Shrine unit creation to that Alterac quest and update the flavour accordingly.

Reply:
Doesn't look resolved!